lib/src/sdk.ts

```diff
@@ -344,12 +347,17 @@ export class ExchangeSDK {
     });
 
     // 3 - Send payload
-    const transaction = await this.walletAPIDecorator.createTransaction({
-      recipient: recipientAddress,
-      amount: fromAmountAtomic,
-      currency,
-      customFeeConfig,
-    });
+    const transaction = await this.walletAPIDecorator
+      .createTransaction({
+        recipient: recipientAddress,
+        amount: fromAmountAtomic,
+        currency,
+        customFeeConfig,
+      })
+      .catch((error: Error) => {
+        this.handleError(error);
+        throw error;
+      });
```

Comment:
Follow what we have in swap, I would say this error is already covered by LL

Reply:
Yes, that was my initial idea. I think we've discussed this before that i should not use the `handleError` method for these two.

Reply:
Errors must be showed always.
Here the errors are already handle by LL, but I do not think they are handle in getSellPayload.